README.rst

```diff
@@ -43,17 +43,30 @@ in *transaction commit time order* and with a guarantee that *no data will be lo
 Installation
 ------------
 
-Prerequisites:
+Prerequisites
+^^^^^^^^^^^^^
 
- #. Python 2.7.*
+ #. Python 2.7.*+
  #. AWS-CLI installed and configured
  #. A PostgreSQL 9.4+ server with logical replication enabled
  #. A Kinesis stream
 
-Install:
+Pre-Install Instructions for Python 3
```

Comment:
If we were to use a branch and not the pypi release, please just use an editable install against git. and place it in the requirements file. No need for special instructions. 

See 4th common case for format: https://pip.readthedocs.io/en/stable/user_guide/#requirements-files

Reply:
Happy to do this but this will move all existing deploys to a non-stable branch of this dependency. It seems more desirable to me to have a temporarily more complex readme and set of installation instructions for python3 users than to potentially put production deploys at risk.

Would you still like me to make this change?

Reply:
I think we should totally do this in requirements. It will allow us to also wire up travisci so we can run python3 tests on this branch. As this is not merging till [#22](https://awslabs/kinesis-aggregation#22) is in pypi this is going to be even more important.

Reply:
Waiting to merge until #22 merges changes a lot here. My point was that I don't think it's a good idea to follow the initial suggestion which was to point this to a development branch of a dependency.

I don't think this needs to wait until #22 is shipped to PyPi as no changes here depend on any change in the underlying library, so I think this delay is manufactured. Though happy to wait if automated travis tests in python 3.3+ are a requirement for this to merge.

Reply:
It is a requirement. The reason for the pull is to fix Issue #10. If at the end of this we don't have automatically deployable and tested python 3 support than what the point.